readme.md

```diff
@@ -77,10 +77,27 @@ user = {
 
 ### Quick Quiz
 
--  What's another "rubyist" way to add items to an array?
--  What is one main difference between Ruby's `hashes` and Javascript's `object literals`?
--  What are some useful methods we can call on collections?
--  Where would I go look if I wanted to find more methods?
+1.  What's another "rubyist" way to add items to an array?
+2.  What is one main difference between Ruby's **hashes** and Javascript's **object literals**?
+3.  What are some useful methods we can call on collections?
+4.  Where would I go look if I wanted to find more methods?
+
+<details>
+  <summary>
+    Answers
+  </summary>
+  <ol>
+    <li> <code> [1,2,3] + [4,5] </code></li>
+    <li> The syntax for referencing properties differs.
+      <ul>
+        <li>JS: <code>user.name</code> or <code>user["name"]</code> </li>
+        <li>Ruby: <code>user[:name]</code> </li>
+      </ul>
+    </li>
+    <li>.push, .pop, .join</li>
+    <li>  https://ruby-doc.org/core-2.2.3/Enumerable.html </li>
+  </ol>
+</details>
 
```

Comment:
^^ In the details summary for the answer to number 4, it'd look nicer to have the text "Ruby Documentation" and then hyperlink